pkgs/by-name/ro/roon-tui/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchFromGitHub
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "roon-tui";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "TheAppgineer";
+    repo = pname;
+    rev = version;
+    hash = "sha256-rwZPUa6NyKs+jz0+JQC0kSrw0T/EL+ms2m+AzHvrI7o=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "roon-api-0.1.1" = "sha256-aFcS8esfgMxzzhWLeynTRFp1FZj2z6aHIivU/5p+uec=";
+    };
+  };
+
+  meta = {
+    description = "A Roon Remote for the terminal";
+    homepage = "https://github.com/TheAppgineer/roon-tui";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ MichaelCDormann ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
Why shouldn't this work on other platforms?

Reply:
From what I understand it probably could, but I can only test and verify on x86_64-linux. I figured if somebody wanted to verify that it works on other platforms and help maintain this derivation for those platforms they can. Is the best approach to go ahead and enable other platforms anyway?

Reply:
The approach is to set this to all platforms that theoretical work, like all linux and then see what other users report and what the CI is telling us.